src/http_client.ts

```diff
@@ -34,6 +38,12 @@ class LogflareHttpClient {
     protected readonly apiKey: string
     protected readonly fromBrowser: boolean
     protected readonly apiBaseUrl: string
+    /**
+     * onError takes in an optional callback function to handle any errors returned by logflare
+     */
+    protected readonly onError?: ((payload: {
+        batch: object[];
+    }, err: Error) => void) | undefined
```

Comment:
typing looks off, should be `null`

Reply:
oops, intended to use `undefined` instead of `null` since this option would default to `undefined` if nothing was passed in

Reply:
i've removed `onError` from the `defaultOptions` since it's not used